README.md

```diff
@@ -105,6 +105,59 @@ steps:
         body: output
       })
 ```
+Specify terraform working directory and previous outputs can be used in subsequent steps for a concise comment on the pull request
+```yaml
+env:
+    tf_actions_working_dir: <Terraform working directory>
+steps:
+- uses: actions/checkout@v2
+- uses: hashicorp/setup-terraform@v1
+
+- name: Terraform fmt
+  id: fmt
+  run: cd ${{ env.tf_actions_working_dir }} && terraform fmt
```

Comment:
GitHub Actions has a working directory option. Cleaner than the `&&` syntax.

```suggestion
  working-directory: ${{ env.tf_actions_working_dir }}
  run: terraform fmt
```

Reply:
To make it even more cleaner
```
defaults:
  run:
    working-directory: ${{ env.tf_actions_working_dir }}
steps:
```